### ISSUE TITLE
Impossible to scroll during serial TTY

### ISSUE THREAD

**Issue by dcsouthwick** [warpdotdev/Warp#2921]
### Discord username (optional)

Wick#2700

### Describe the bug

Warp does not make it obvious how to scroll up during a serial (TTY) connection, which is supported on every other terminal emulator (and a critical tool as serial devices tend to spit a lot of lines rapidly!) 
It would be nice to implement at least minimal functionality found in other terminals like:
- make scrollbar visible
- scroll with mousewheel
- scroll in steps with ⌘+↑  or ⌘+↓

### To Reproduce

1. Open some serial connection with warp (eg. `screen /dev/tty.usbserial-* 115200`).
2. You are now locked in a screen that you cannot scroll. Any lines that exceed the viewable area are seeming inaccessible or lost.

### Expected behavior

scroll up or down in some way to view previous lines beyond the current focus

### Operating System

MacOS

### OS Version

13.3

### Shell Version

zsh 5.9 (x86_64-apple-darwin22.0)

### Warp Version

v0.2023.04.04.08.03.stable_01

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
Hi @dcsouthwick I'll bring this up with the team and we'll update this thread. 

Unfortunately, I don't have a serial device I can test this with so we may need some additional info from you such as [logs](https://docs.warp.dev/help/sending-us-feedback#gathering-warp-logs).

**Comment by dmachiavello** [comment]
+1 for this. Anytime i'm using ```screen``` on a server i lose all ability to scroll through history. this feature is a must since otherwise it's unusable for monitoring long running processes and i have to switch back to iterm2 just for this feature alone.

**Comment by dmachiavello** [comment]
@dannyneira Is there any movement on this? i use `screen` too much to make Warp my primary terminal.

**Comment by dannyneira** [comment]
Sorry about this, I was able to reproduce the issue and we'll be tracking it in #2599. As a possible workaround, I was able to get scrolling working by enabling mouse reporting for `screen` with the following line in the `~/.screenrc` file ```termcapinfo xterm* ti@:te@``` as well as [Mouse and Scrolling Reporting enabled](https://docs.warp.dev/features/full-screen-apps#mouse-and-scroll-reporting) in the Warp Settings > Features.

Closing as duplicate since there is already a ticket tracking this issue of scrolling within `screen`.
